src/components/FlowList.vue

```diff
@@ -81,43 +107,46 @@
 
   const DEFAULT_LIMIT = 40
 
-  const search = ref<string>('')
-  const searchDebounced = useDebouncedRef(search, 800)
+  const headerExpanded = ref(false)
+  const selected = ref<string[]>([])
 
-  const page = ref(1)
-  const offset = computed({
-    get: () => (page.value - 1) * DEFAULT_LIMIT,
-    set: (value: number) => {
-      page.value = Math.ceil(value / DEFAULT_LIMIT) + 1
+  const search = ref<string>()
+  const searchDebounced = useDebouncedRef(search, 800)
+  const nameLike = computed({
+    get() {
+      return searchDebounced.value === '' ? undefined : searchDebounced.value
+    },
+    set(value: string | undefined) {
+      searchDebounced.value = value
     },
```

Comment:
This prevents the `nameLike` filter from being applied to the query params when it's empty - follow up from https://github.com/PrefectHQ/vue-compositions/pull/243

Reply:
It also ensures that an empty `nameLike` param doesn't incorrectly cause `isCustomFilter` to be `true`